app/views/govuk_component/docs/previous_and_next_navigation.yml

```diff
@@ -11,6 +11,25 @@ body: |
   - a label that can add extra info (ie page number) that will be displayed under the title
 
   If one of the 2 parameters is nil, no link will appear.
+accessibility-criteria: |
+  The component must:
+
+  - identify itself as pagination navigation
+
+  Links in the component must:
+
+  - include rel next or rel previous
```

Comment:
This seems like an implementation detail, is it used in an accessibility context?